src/main/java/teammates/ui/controller/InstructorFeedbackAbstractAction.java

```diff
@@ -46,17 +56,17 @@ protected FeedbackSessionAttributes extractFeedbackSessionData(boolean isCreatin
                 .build();
 
         String paramTimeZone = getRequestParamValue(Const.ParamsNames.FEEDBACK_SESSION_TIMEZONE);
-        if (paramTimeZone != null) {
-            try {
-                attributes.setTimeZone(ZoneId.of(paramTimeZone));
-            } catch (DateTimeException e) {
-                log.warning("Failed to parse time zone parameter: " + paramTimeZone);
-            }
+        Assumption.assertPostParamNotNull(Const.ParamsNames.FEEDBACK_SESSION_TIMEZONE, paramTimeZone);
```

Comment:
Hmm. From the comments at the beginning of this function: this field can be `null` when editting feedback session. Yet another problem: when we create a `FeedbackSessionAttributes` it will always have a non-null time zone. So if the user doesn't want to change the timezone, the backend actually resets the timezone to default.